### ISSUE TITLE
Add binary releases for jsonnet-deps and jsonnet-lint (perhaps put all tools together in the same archive?)

### ISSUE THREAD

**Issue by sbarzowski** [google/go-jsonnet#480]
We already have binary releases for the interpreter binary and for the formatter. We have 2 new tools now, so we should include them.

**Comment by iwoloschin** [comment]
Would it be possible to get python bindings for these (and `jsonnetfmt` too!) as well?

**Comment by sbarzowski** [comment]
@iwoloschin It's possible, but it would require some work. If we do so, it should probably be a separate python package. We'd probably also need to create C bindings first as an intermediate step. If someone wants to picks this up, I'll be happy to help.

For now, when you need this functionality from Python, I would recommend just using the commands. This approach has some additional advantages actually, like allowing the end-user to control the version that is used.

**Comment by iwoloschin** [comment]
@sbarzowski yeah, I'm not sure I *need* this functionality and it is certainly easy enough to just call a binary, it just sort of seems like a logical next step.

**Comment by johnbartholomew** [comment]
Pre-built binaries for jsonnetfmt, jsonnet-deps, jsonnet-lint were added to the Go Releaser config a while ago.
